docs/kinto-admin.rst

```diff
@@ -0,0 +1,262 @@
+.. _admin-ui: 
```

Comment:
A good practice is to keep main page anchor in sync with file name (here `kinto-admin` instead of `admin-ui`)